src/DistIL/AsmIO/Entities/MethodDef.cs

```diff
@@ -20,6 +20,7 @@ public abstract class MethodDesc : MemberDesc
     public bool IsPublic => (Attribs & MethodAttributes.MemberAccessMask) == MethodAttributes.Public;
 
     public bool IsConstructor => Name == ".ctor";
+    public bool IsStaticConstructor => Name == ".cctor";
     public bool IsDestructor => Name == "Finalize";
```

Comment:
I'm just remembering this, but these should probably also check `attribs.HasFlag(SpecialName)` for correctness